ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__multi-nightly-4.16-upgrade-from-stable-4.15.yaml

```diff
@@ -4,7 +4,7 @@ base_images:
     namespace: ocp
     tag: ansible
   cli:
-    name: "4.15"
+    name: "4.16"
```

Comment:
Just like what we discussed in slake, some steps in install workflow are using `from: cli`, so I think the cli should be aligned with the source cluster version, and once that, it also keep consistent with the existing ci jobs.